### ISSUE TITLE
Add the "anyl" option to the "all in" block

### ISSUE THREAD

**Issue by AlexITMan** [thunkable/thunkable-issues#195]
![изображение](https://user-images.githubusercontent.com/20015046/61438359-f0d87300-a947-11e9-91d3-f3e9cfd3f2d7.png)

Now to create a general list of all available components, you will have to create a list and add all the "all in" blocks to it.

![изображение](https://user-images.githubusercontent.com/20015046/61438672-a7d4ee80-a948-11e9-8278-9f5ac776396d.png)

Of course, not all of these components will be located on the same screen, but a loop according to one list will be more convenient to use several loops if you need to look for components of different types.

**Comment by mark-friedman** [comment]
I'm not sure how many situations need this capability but it might be useful to add it.  One thing to note, though, is that it would be hard currently to do anything useful with such a list of all components because we have no way to get the type of a component.

**Comment by AlexITMan** [comment]
Is it possible to create and in the component constructor to initialize the read-only "type" property, which will contain the component type. For example, "Button" for a button, etc.

When we add a component to the application screen, a specific type of component is created. If I drag the button icon, a button appears on the screen, not a label or switch. That is, a component of a certain type is created. Do I understand correctly or is everything more complicated?

**Comment by mark-friedman** [comment]
@AlexITMan Yes, it would be possible to add another block which returned the type of a component.  That could then be used in the blocks that you prosed in issues #196 and #199.  I would suggest adding another feature request for creating a block which returns the type of a component.
